tests/provider/dwd/observation/test_io.py

```diff
@@ -239,7 +239,7 @@ def test_export_spreadsheet(tmpdir_factory):
 
     df = request.values.all().df
 
-    filename = tmpdir_factory.mktemp("data").join("observations.xlsx")
+    filename = tmpdir_factory.mktemp("data").join("observation.xlsx")
```

Comment:
I think it is fine to use the plural form for naming the output file throughout the whole file.